src/build.sh

```diff
@@ -147,23 +164,91 @@ function parse_build_options()
         shift
         ;;
       --menu | -n)
-        options_values['MENU_CONFIG']="${configurations[menu_config]:-nconfig}"
+        options_values['MENU_CONFIG']="${configurations[menu_config]:-"nconfig"}"
         shift
         ;;
       --doc | -d)
-        options_values['DOC_TYPE']="${configurations[doc_type]:-htmldocs}"
+        options_values['DOC_TYPE']="${configurations[doc_type]:-"htmldocs"}"
```

Comment:
This is a great change, but can we create a separate commit for this change and the one in the `--menu`? I'm asking because this is not part of this pkg change.